README.md

```diff
@@ -7,39 +7,38 @@ pip install marine
 ```
 
 ## Usage
-1. 
-    Initialize an instance of Marine (note that only one can be initialized per process):
-    ```python
-    marine_instance = Marine('path to libmarine.so')
-    ```
-   
-2.
-    Call the method `filter_and_parse` with a BPF, a display filter and fields to parse.
-    Every parameter is optional - if it's passed as `None` then it won't be applied to the packet, but at least one parameter is required. 
-    ```python
-    passed, output = marine_instance.filter_and_parse(b'your packet', 'ip', 'tcp', ['ip.src', 'ip.dst'])
-   ```
-   
-3. 
-    `passed` is a boolean inidicating whether or not the packet passed the filter. 
-    If a filter wasn't supplied, it will be `True`.
-
-    `output` is a dict between field name to its parsed value.
-     If the packet didn't pass the filter or fields weren't passed to the function, it will be `None`.
-    
+```python
+import marine
+
+passed, result = marine.filter_and_parse_packet(b"your packet", "ip host 1.1.1.1", "tcp.port == 80", ["ip.src", "ip.dst"])
+if passed:
+    print(f"{result['ip.src']} -> {result['ip.dst']}")
+
+# Also available
+marine.parse_packet(...)
+marine.filter_packet(...)
+marine.validate_fields(...)
+marine.validate_display_filter(...)
+marine.validate_bpf(...)
+```
+For advanced usages (not recommended) see the `get_marine()` function and the `Marine` class    
 
 ## Contributing
 ### Guidelines
 Syntax formatting is done using [Black](https://github.com/psf/black)
 
 ### Running Tests
-The tests are written using pytest. To run the tests, you need to provide the library file (`libmarine.so`),
-and place it next to the file `tests/fixtures/marine/marine_fixtures.py`.
+The tests are written using pytest. To run the tests, you need to provide the library file (`libmarine.so`) and its dependencies.
+`marine` expects `libmarine.so` to reside in `marine/.wslibs`.
 
 Then, simply run `tox`.
 
 Additionally, syntax is checked with flake8 by running `flake8 marine tests` from the root directory of the project.
 
+### Packaging
+An `x86_64` `manylinux2014` wheel is built by our CI, based on a `manylinux` image supplied by `marine-core` and patched to meet our needs.
+There is currently no support for installing on other platfroms
```

Comment:
How hard would it be to add support for other platforms? Like Windows for example?

Reply:
Dude are you nuts

Reply:
Windows is problematic for 2 reasons:
1. We have experience building for this target
2. There's no windows equivalent for `auditwheel` AFAIK.
Either way that's pretty much out of scope